prow/prow.yaml

```diff
@@ -496,7 +496,7 @@ metadata:
   name: "deck"
 ---
 kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v11
```

Comment:
v11 ? isn't it a typo and should it be `v1` instead ? 😝

Reply:
🤣 yeah the current version was very old :P

Nice catch, thanks